src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -78,129 +80,35 @@ const ApplicationDetails: React.FC = () => {
-            </VStack>
-          </HStack>
+        <VStack align="start" spacing={4} p={2} bg="gray.50">
+          {applicantData?.map((item) => (
+            <HStack
+              spacing={8}
+              w="100%"
+              // boxShadow="0px 4px 4px 0px #00000040"
+              p="2"
+              borderRadius="md"
+              bg="white"
+            >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)